packages/client/hmi-client/src/utils/pivot.ts

```diff
@@ -1,12 +1,14 @@
 // Create pivot table
 import _ from 'lodash';
+import { Model } from '@/types/Types';
 
 export interface PivotMatrixCell {
 	row: number;
 	col: number;
 	rowCriteria: any;
 	colCriteria: any;
-	value: any;
+	controllers?: string[];
+	content: any;
 }
```

Comment:
I don't like the idea of introducing domain/business specific logic into utils - ideally what I want for util is to be agnostic to both framework and specific domains, so they are true-utilities that can be reused with relative ease.

`controllers` is quite specific to these types of ODE-problems with topological contraints.

Let's maybe talk a bit about what we really need and see if we can make this a bit more flexible.